tests/test_orm.py

```diff
@@ -27,6 +29,22 @@ class Meta:
         mock_api.endpoint.assert_called_with('test/123')
         assert test.name == 'alice'
 
+    def test_refresh_with_empty_id(self):
+        mock_api = mock.MagicMock()
+        mock_api.endpoint.return_value.get.return_value.content.data = data.JsonApiObject(
```

Comment:
This shouldn't be needed now when the code protects against calling the API with invalid parameters.